app/views/posts/index.html.erb

```diff
@@ -1,3 +1,54 @@
-<h1>Input your favourite RSS feed</h1>
 
-<%= render 'form' %>
+<div class="container">
+
+<div class="page-header">
+<h1>Share or Schedule</h1>
+</div>
+
+<div class="row row-flex row-flex-wrap">
+
+	<% @urls.find_each do |url| %>
+       <% @feed = Feedjira::Feed.fetch_and_parse url.url %>
+        <% @entries = @feed.entries %>
```

Comment:
.entries is a feedjira method. I called the instance variable @entries because it aligns with the the .entries method. 

I am not sure why you are getting that error. I haven't seen the same. We can look at it together when you get in.